app/src/main/kotlin/com/wire/android/ui/home/conversations/details/participants/GroupConversationParticipantsViewModel.kt

```diff
@@ -76,23 +71,25 @@ open class GroupConversationParticipantsViewModel @Inject constructor(
         navigationManager.navigateBack()
     }
 
-    fun openProfile(participant: UIParticipant) = viewModelScope.launch {
-        if (participant.isSelf) navigateToSelfProfile()
-        else if (participant.isService && participant.botService != null) navigateToServiceProfile(participant.botService)
-        else navigateToOtherProfile(participant.id)
+    fun openProfile(participant: UIParticipant, direction: Direction? = null) = viewModelScope.launch {
```

Comment:
I guess this `direction` parameter was added only to be able to test it, right?
I don't think it's the best approach and it can be tested well even without it. So the problem was probably that `OtherUserProfileScreenDestination` created inside this method is different than one used to `coVerify` and it results a false, right?
Then you can always compare the arguments this way:
```
        coVerify(exactly = 1) {
            arrangement.navigationManager.navigate(
                    withArg {
                        assertTrue(it.destination.route == otherUserProfileScreen.route)
                    }
            )
        }
```